demo/index.html

```diff
@@ -33,7 +33,12 @@
   <style is="custom-style" include="demo-pages-shared-styles">
     div[role="listbox"] {
       border: 1px solid #e5e5e5;
+      @apply(--layout-vertical);
     }
+    paper-item-body {
+      height: 1px;
```

Comment:
Shouldn't this be applied to 'paper-item-body' element from 'paper-item-body.html' instead fixing in Demo? Because, I guess it's not only Demo issue, it appears in my app also.